Admin/AdminHelper.php

```diff
@@ -284,13 +285,11 @@ public function getElementAccessPath($elementId, $entity)
                     $partReturnValue .= $dot.$id;
                     $dot = '.';
                     $id = '';
-                } else {
-                    $dot = '';
                 }
             }
 
-            if ($dot !== '.') {
-                throw new \Exception(sprintf('Could not get element id from %s Failing part: %s', $elementId, $subValue));
+            if (!empty($id)) {
```

Comment:
> I prefer not to add comments to the code, but if necessary I will do it...

You're probably referring to the saying that goes something like this:

> If you need to comment your code, you failed to make it self-explanatory

In this instance, the code was failed to be made self-explanatory AND the author failed to admit his failure and make a comment.

So starting to make the code more self-explanatory is indeed the way to go, but in the end, if you feel a comment would not hurt, please don't refrain from adding one.